site/content/en/docs/Installation/install_om.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Install Open Match"
+linkTitle: "Install Open Match"
+weight: 2
+description: >
+  Steps to install Open Match in a kubernetes cluster.
+---
+
+Open Match comprises of a set of services that enable core functionality such as Ticket management, filtering, Match generation etc. It also includes a
+state storage that Open Match uses to persist state needed for its functioning.
+
+The simplest way to install Open Match is to use the install.yaml files for the latest release.
+This installs Open Match with the default configuration.
+
+```bash
+# Create a cluster role binding (if using gcloud on Linux or OSX)
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user `gcloud config get-value account`
+
+# Create a cluster role binding (if using gcloud on Windows)
+for /F %i in ('gcloud config get-value account') do kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user %i
+
+# Create a cluster role binding (if using minikube)
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --serviceaccount=kube-system:default
+
+# Create a namespace to place all the Open Match components in.
+kubectl create namespace open-match
+
+# Install the core Open Match services and monitoring services.
+kubectl apply -f https://open-match.dev/install/v{{ .Site.Params.release_version }}/yaml/install.yaml --namespace open-match
```

Comment:
Make sure this parameter renders. It didn't on the live version of development.open-match.dev. It might be because it's in a "pre" block.

Reply:
Is it possible to introduce a different parameter for yaml file location?  That way the in development docs could point to the files generated by post-submit?

Reply:
I ve just restructured what was there previously - Was gonna defer this to @Laremere to actually populate this page / demo instructions page. Lets have this in for now to check in the structure (since this is already existing anyways) - and then we can figure out this bit.